docs/contributing/gsoc/2021/index.rst

```diff
@@ -9,12 +9,12 @@ https://summerofcode.withgoogle.com/how-it-works/#timeline
 About DFFML
 -----------
 
-DFFML is a plugin based library / framework for machine learning. It allows
+DFFML is a plugin-based library/framework for machine learning. It allows
 users to wrap high or low level implementations of models that use various
-machine learning libraries, so as to interact will lots of different model
+machine learning libraries, so as to interact will lot of different model
 implementations in the same way.
 
-DFFML is also a tool for dataset generation. DFFML ues directed graphs to
+DFFML is also a tool for dataset generation. DFFML use directed graphs to
```

Comment:
Shouldn't it be `uses` instead of `use` ?

Reply:
okay

Reply:
okay will do that